scenario/ops_main_mock.py

```diff
@@ -96,6 +97,9 @@ def setup_framework(
     )
     debug = "JUJU_DEBUG" in os.environ
     setup_root_logging(model_backend, debug=debug)
+    # ops sets sys.excepthook to go to Juju's debug-log, but that's not useful
```

Comment:
this sounds familiar. are you sure about this change? How is it related to container execs?

Reply:
Ah, this is from another PR. I might have branched this off that branch instead of main by mistake. That one has been merged, so this should go away if I rebase. If not I'll remove it for clarity, sorry.